learning_journal/static/app.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+    form = $("#new_entry_form");
+    form.submit(function(e){
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/',
```

Comment:
Why point this at your home url?  AJAX allows you to send a request anywhere you want.